src/test/groovy/com/github/benmanes/gradle/versions/DependencyUpdatesSpec.groovy

```diff
@@ -83,14 +83,14 @@ final class DependencyUpdatesSpec extends Specification {
     }
   }
 
-  @Unroll('Single project with no repositories (#outputFormatter)')
-  def 'Single project with no repositories'() {
+  @Unroll
```

Comment:
Use `Unroll` variables in the method name, remove redundant wording in `Unroll` annotation.